fix(message): fix some messages

fujicoin opened the pull request:
Fixed a bug related to TR_TRANSACTIONS_SEARCH_TIP.

fujicoin commented:
Attach the correct translation source file (master.json) for crowdin.con after making this modification. Please upload to crowdin.

master.json.zip -- deleted

fujicoin commented:
As for `SETTINGS_ADV_COIN_CONN_INFO_NO_CONNECTED`, the context has changed, so I changed the message appropriately.
Updated master.json.

[master.jason.zip](https://github.com/trezor/trezor-suite/files/7947062/master.jason.zip)

jerechinsky commented:
Great catch, @fujicoin! I have adjusted the English copy in Crowdin according to your suggestions. 

I haven't addressed it previously, but we don't change the English copy via editing `messages.ts` directly. Instead, our copywriters use the English translation in Crowdin to change the copy. I know you previously raised some concerns about this workflow, mainly how translators might not see the updated English copy (which is a legitimate concern). 

However, we have mitigated this by regularly doing so-called `backports` when @mroz22's script replaces original default messages with approved "translations" of English from Crowdin. After doing a backport, each updated default message will un-approve itself in Crowdin for all the other languages, so translators will see the change and will be able to fix it.

tomasklim commented on packages/suite/src/support/messages.ts:
```diff
@@ -6693,7 +6693,7 @@ export default defineMessages({
         id: 'TR_TRANSACTIONS_SEARCH_TIP_10',
         defaultMessage:
-            'Tip: You can combine AND (&) and OR (|) operators for more complex searches. For example For example <strong>> 2020-12-01 & < 2020-12-31 | > 2020-12-01 & < 2020-12-31</strong> will show all all transactions in January 2020 or December 2020.',
+            'Tip: You can combine AND (&) and OR (|) operators for more complex searches. For example For example <strong>> 2020-01-01 & < 2020-01-31 | > 2020-12-01 & < 2020-12-31</strong> will show all all transactions in January 2020 or December 2020.',
     },
```
```suggestion
            'Tip: You can combine AND (&) and OR (|) operators for more complex searches. For example <strong>> 2020-01-01 & < 2020-01-31 | > 2020-12-01 & < 2020-12-31</strong> will show all all transactions in January 2020 or December 2020.',
    },
```

tomasklim replied:
@alex-jerechinsky there is `For example` twice.

jerechinsky replied:
thank you, fixed in crowdin

fujicoin commented:
@tomasklim 
that's right. Fixed.

fujicoin commented:
> I haven't addressed it previously, but we don't change the English copy via editing `messages.ts` directly. Instead, our copywriters use the English translation in Crowdin to change the copy. I know you previously raised some concerns about this workflow, mainly how translators might not see the updated English copy (which is a legitimate concern).
> 
> However, we have mitigated this by regularly doing so-called `backports` when @mroz22's script replaces original default messages with approved "translations" of English from Crowdin. After doing a backport, each updated default message will un-approve itself in Crowdin for all the other languages, so translators will see the change and will be able to fix it.

@alex-jerechinsky 
So that's it. A copywriter who is not a programmer is writing English. So I understood that it was a complicated workflow.
I refer to the code of the program for the added / changed keys, start the Web version of Trezoe Suite, and check the validity of the translation in the actual usage as much as possible.
I myself can adapt to your workflow, but I think it's probably difficult for ordinary translators to keep up with.

fujicoin commented:
I have read backport-en.ts and have a complete understanding of how backport works.
https://github.com/trezor/trezor-suite/blob/develop/packages/suite-data/scripts/backport-en.ts

If I find an error in the future, I should send a PR of the fix to en.json.
But en.json doesn't contain all the keys. Therefore, in the case of this TR_TRANSACTIONS_SEARCH_TIP related modification, there is no choice but to send a PR to modify messages.ts.

fujicoin commented:
@alex-jerechinsky
I thought about the backport workflow.
The next time you backport, why not upload master.json as a source file to crowdin and at the same time upload it to en translation?
Then en.json will have all the keys. After that, if you modify en and it is approved, you can immediately rename it to master.json and upload it to crowdin.
Isn't this method surely speeding up the update of the crowdin source file? And when I point out a mistake, I only have to do it for en.json.
This means that we don't have to worry about the defaultMessage description in the messages.ts file. Still, sometimes it's better to work on the backport.

fujicoin commented:
I've seen the messages.ts and crowdin source files modified around the next commit.
https://github.com/trezor/trezor-suite/commit/7f81d831608cf02cb37ea8da02d34fb6bc4f2882

This time en.json has become much smaller, so it's hard to see what's changed.
Therefore, I added and modified the Japanese translation of crowdin following the changes in messages.ts.

Key with changed wording:
MODAL_ADD_ACCOUNT_DESC
RECIPIENT_REQUIRES_UPDATE
TR_SUITE_WEB_LANDING_DEPRECATION_BANNER_PARAGRAPH_5
TR_TROUBLESHOOTING_TIP_BRIDGE_INSTALL_TITLE

Key name changed:
TR_DEVICE_SETTINGS_HOMESCREEN_IMAGE_SETTINGS
→ TR_DEVICE_SETTINGS_HOMESCREEN_IMAGE_SETTINGS_TT
     
Currently the Japanese translation is perfect.
This PR is now closed.

fujicoin commented:
At the moment, there is no difference between the wording of en.json and messages.ts.
Therefore, en.json can be emptied.
